src/db/models/imagestore.js

```diff
@@ -0,0 +1,25 @@
+const { Model } = require("sequelize");
+
+module.exports = (sequelize, DataTypes) => {
+  class Imagestore extends Model {
+    static associate(models) {
+      Imagestore.belongsTo(models.Salon, {
+        foreignKey: "salonId",
+        onDelete: "CASCADE",
+      });
+    }
+  }
+
+  Imagestore.init(
+    {
+      salonId: DataTypes.UUID,
```

Comment:
it would be better to add references model.
```
salonId: {
        type: DataTypes.UUID,
        references: {
          model: "salons",
          key: "salonId",
        },
        onUpdate: "CASCADE",
        onDelete: "CASCADE",
      }
```